### DISCUSSION TITLE
Regex to match Ubuntu image tags eg jammy-20220815

### DISCUSSION THREAD

**Discussion by cdloh** [renovatebot/renovate#25001] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate auto updates Ubuntu Image tags

### What you tried so far.

I've tried using the "Ubuntu" versioning but it doesn't support the image tags as per the documentation.

Was hoping I would be able to do it via a regex matcher but can't get that to work either.

If it's not possible all good.

**Comment by rarkins** [comment]
What's an example before/after upgrade?

**Reply by cdloh** [reply]
eg  `jammy-20230816` -> `jammy-20230916` 

Ideally it would be able to detect the major change as well (ie the codename bump) but that would require support within renovate I believe.

**Reply by rarkins** [reply]
It works with the new versionCompatibility feature, see https://github.com/renovate-reproductions/jamm2

It should be live in the hosted app tomorrow

**Reply by cdloh** [reply]
Argh I created an MR to kinda do this prior to reading this update see https://github.com/renovatebot/renovate/pull/25004
